insights/parsers/kpatch_list.py

```diff
@@ -0,0 +1,67 @@
+"""
+KpatchList - command ``/usr/sbin/kpatch list``
+==============================================
+
+The ``/usr/sbin/kpatch list`` command provides information about
+the installed patch modules.
+
+Sample content from command ``/usr/sbin/kpatch list`` is::
+    Loaded patch modules:
+    kpatch_3_10_0_1062_1_1_1_4 [enabled]
+
+    Installed patch modules:
+    kpatch_3_10_0_1062_1_1_1_4 (3.10.0-1062.1.1.el7.x86_64)
+
+Examples:
+    >>> 'kpatch_3_10_0_1062_1_1_1_4' in kpatchs.loaded
+    True
+    >>> kpatchs.loaded.get('kpatch_3_10_0_1062_1_1_1_4')
+    'enabled'
+    >>> 'kpatch_3_10_0_1062_1_1_1_4' in kpatchs.installed
+    True
+    >>> kpatchs.installed.get('kpatch_3_10_0_1062_1_1_1_4')
+    '3.10.0-1062.1.1.el7.x86_64'
+"""
+
+from insights.specs import Specs
+from insights.parsers import SkipException, ParseException
+from insights import parser, CommandParser
+
+
+@parser(Specs.kpatch_list)
+class KpatchList(CommandParser):
+    """Class for command: /usr/sbin/kpatch list"""
+    def parse_content(self, content):
+        if not content:
+            raise SkipException("No Data from command: /usr/sbin/kpatch list")
+
+        self._loaded = {}
+        self._installed = {}
+        cur_dict = {}
+        for line in content:
+            if not line.strip():
+                continue
+
+            if 'Loaded patch modules' in line:
+                cur_dict = self._loaded
+                continue
+
+            if 'Installed patch modules' in line:
+                cur_dict = self._installed
+                continue
+
+            kpatch, info = [k.strip('()[]') for k in line.split()]
```

Comment:
I have got the following output of `kpatch list`

~~~
# kpatch list
Loaded patch modules:
kpatch_7_0_1_el7
Installed patch modules:
kpatch-7-0-1-el7.ko (3.10.0-121.el7.x86_64)
kpatch-7-0-1-el7.ko (3.10.0-123.el7.x86_64)
~~~

This line will fail for the loaded part.